src/job-posting/dto/job-query.dto.ts

```
@@ -0,0 +1,81 @@
+  @IsOptional()
+  @IsBoolean()
+  @Transform(({ value }) => value === "true" || value === true)
+  isFeatured?: boolean
+
+  @IsOptional()
+  @IsString()
+  skills?: string
+
+  @IsOptional()
+  @IsString()
+  sortBy?: string = "createdAt"
```

ESLint

[error] Replace `"createdAt"` with `'createdAt';`

(prettier/prettier)